server/routerlicious/packages/services-shared/src/storage.ts

```diff
@@ -197,8 +198,10 @@ export class DocumentStorage implements IDocumentStorage {
             isSessionAlive: true,
         };
 
-        winston.info(`Session: ${JSON.stringify(session)}`, { messageMetaData });
-        Lumberjack.info(`Session: ${JSON.stringify(session)}`, lumberjackProperties);
+        winston.info(`Create session with enableDiscovery flag as ${enableDiscovery}: ${JSON.stringify(session)}`,
```

Comment:
>`Create session with enableDiscovery flag as ${enableDiscovery}: ${JSON.stringify(session)}` [](http://example.com/codeflow?start=21&length=92)

(nit) use a variable to avoid the duplications.

Reply:
Got it. Thanks. Done. :)